### ISSUE TITLE
Causes error if user model is different name from 'Users'

### ISSUE THREAD

**Issue by t-kinoshita** [Beskhue/CookieTokenAuth#4]
I'm trying to use CookieTokenAuth in some CakePHP3 project, where user data table is not `users`.

In Current implementation, though, `users` model is hard-coded, and specifying `userModel` doesn't work well.

```php
// AuthTokensTable.php
public function initialize(array $config)
{
    $this->addBehavior('Timestamp');

    $this->belongsTo('Users');
}

// CookieTokenAuthenticate.php
$tokenEntity = $authTokens->findBySeries($series)->contain('Users')->first();
```

It caused the error, `Error: [Cake\Database\Exception] SQLSTATE[42S02]: Base table or view not found: 1146 Table 'project_name.users' doesn't exist`.

**Comment by t-kinoshita** [comment]
The last line of `CookieTokenAuthenticate#getUserFromCookieData` is `return $this->_findUser($user->username);`, the property name 'username` is also hard coded.

**Comment by tomcur** [comment]
Thanks for the issue report. I am quite busy right now, but I will have a look at this next week.

**Comment by tomcur** [comment]
Thank you for your patience.

I have had a look at this issue. You seem to be using version 0.2.1 or older. I had not yet released a new version, though in the master branch custom configurations were supported already. I have now released this enhancement as version 0.3.0.  

Testing, the resulting configuration with `Members` as the user model and `nick` as the username field works fine in version 0.3.0:

```
	$this->loadComponent('Auth', [
            'authenticate' => [
                'Beskhue/CookieTokenAuth.CookieToken' => [
                    'fields' => ['username' => 'nick'],
                    'userModel' => 'Members'
                ],
                'Form' => [
                    'fields' => ['username' => 'nick'],
                    'userModel' => 'Members'
                ],
            ]
	   ]);
```
